tests/Serializer/BaseSerializationTest.php

```diff
@@ -2119,8 +2169,14 @@ static function (DeserializationVisitorInterface $visitor, $data, $type, Context
 
         $this->dispatcher = new EventDispatcher();
         $this->dispatcher->addSubscriber(new DoctrineProxySubscriber());
+        $this->dispatcher->addSubscriber(new EnumSubscriber());
 
         $builder = SerializerBuilder::create($this->handlerRegistry, $this->dispatcher);
+
+        if (PHP_VERSION_ID >= 80100) {
```

Comment:
Just wonder if we should move this PHP version check to `enableEnumSupport` to be sure that it will not fail on lower versions. 🤔 But on the other hand it will be better to fail fast.

Reply:
done